src/promptflow/promptflow/executor/_script_executor.py

```diff
@@ -51,36 +48,37 @@ def exec_line(
         run_id: Optional[str] = None,
         **kwargs,
     ) -> LineResult:
+        if "line_number" in inputs:
+            inputs.pop("line_number")
```

Comment:
What if the function has a parameter "line_number"?

Reply:
Refined. Only pop line_number if it doesn't exist in inputs.